pkg/kudoctl/cmd/install/install.go

```diff
@@ -171,6 +178,28 @@ func installOperator(operatorArgument string, options *Options) error {
 	return nil
 }
 
+func validateCrds(crds *repo.PackageCRDs, skipInstance bool) error {
```

Comment:
wouldn't it make sense to remove the `skipInstance` and not call this function if `skipInstance` is true?

Reply:
then the caller would know the implementation details of that function - it would know that it performs only certain validation that does not make sense when instance is not installed... That was at least my reasoning behind choosing this path

Reply:
I think a better route for this would be:

1. Create a Validator interface with a Validate function
2. Add Validate functions to each CRD type
3. Validate CRDs ranges through and calls each.

That way, skipInstance only means "do not include the Instance in the list of CRDs".

That may be a larger refactor than we want here, wdyt?

Reply:
agree with gerred on all accounts... 1) love the validate interface idea 2) probably bigger than what was planned here.    we should create an issue with that detail and move forward with this as is.

Reply:
I disagree with `skipInstance` position however...  the position you are taking is that you want logic encapsulated.. that the caller wouldn't know the impl details.     how does that make sense if the caller is providing a boolean on if something should be skipped.   The caller is aware.

Reply:
https://github.com/kudobuilder/kudo/issues/528

Reply:
I mean skipInstance as a concept, not skipInstance in it's existence here. :)